integreat_cms/cms/views/bulk_action_views.py

```diff
@@ -518,3 +518,75 @@ def post(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
             status.DRAFT,
         )
         return super().post(request, *args, **kwargs)
+
+
+class BulkDeletingView(BulkActionView):
+    """
+    Bulk action to delete multiple entries at once
+    """
+
+    def get_permission_required(self) -> tuple[str]:
+        r"""
+        This method overwrites get_permission_required()
+
+        :return: The needed permission to delete entries
+        """
+        return (f"cms.delete_{self.model._meta.model_name}",)
+
+    def post(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
+        r"""
+        Function to delete multiple entries at once
+
+        :param request: The current request
+        :param \*args: The supplied arguments
+        :param \**kwargs: The supplied keyword arguments
+        :return: The redirect
+        """
+        delete_sucessful = []
+        delete_failure = []
+        can_delete = True
+
+        for content_object in self.get_queryset():
+            if hasattr(content_object, "can_be_deleted"):
+                can_delete, error_msg = content_object.can_be_deleted()
```

Comment:
`error_msg` may be re-witten by the next objects. 

Senario: First language was not deletable, the next one is deletable. `error_msg` is once set to the warning message but overwritten by an empty string by the second object. How to reproduce: choose English and Greek in Augsburg and bulk delete